igor-web/src/main/java/com/netflix/spinnaker/igor/travis/client/model/v3/V3Log.java

```diff
@@ -58,10 +58,20 @@ public void setLogParts(List<V3LogPart> logParts) {
         this.logParts = logParts;
     }
 
+    public boolean isReady() {
+        if (logParts == null || logParts.isEmpty()) {
+          return false;
+        }
+        int numberOfParts = logParts.size() - 1;
+      V3LogPart lastLogPart = logParts.get(numberOfParts);
+      return numberOfParts == lastLogPart.number && lastLogPart.isFinal();
+    }
+
     @JsonIgnoreProperties(ignoreUnknown = true)
     static class V3LogPart {
         private String content;
         private Integer number;
+        private boolean _final;
```

Comment:
Generally I'd prefer the jackson annotation to map a `isFinal` attribute here to `final` in the response payload (assuming that the `_` is because of `final` being a reserved word)

Reply:
Better?

Reply:
LGTM